### PR TITLE
Fix/migrations

### PR DESCRIPTION
**Pull request by axefx**
# Seeds database with training data

setups migrations
updates schema and models to match the database

## PR Creator Checklist

- [ ] I wrote a label and description of the requested changes
- [ ] PR was sent to multiple reviewers, including at least one team member and the TPL
- [ ] I performed a self-review of my own code (followed the code review checklist)
- [ ] I linked to the user story/Trello checklist item corresponding to the PR

## Code Review Checklist

Bearing in mind the [Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da):

- [ ] Atomic, well-named commits
- [ ] PR documents the motivation, user story and changes
- [ ] Code is consistently formatted 
- [ ] PR is appropriately scoped- small or only one issue
- [ ] Code is idiomatic and conforms to industry standards
- [ ] At least one team member or the TPL has reviewed the PR and left appropriate comments
- [ ] (When applicable) Reviwer suggestions were acted upon

### CODE REVIEW

**Review by maiali13 — COMMENTED** [review]
glad to see the readme for sbsetup
is there a reason we are deleting notebooks/Police_Violence_Data.ipynb ? are we deleting notebooks? 
Also do we need training_data2.csv?

**Review by maiali13 — APPROVED** [review]
clever fix to get multiple tags displayed in the incidents endpoint

File: dbsetup/seed/evidences.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    # evidence: id, link
+    with open('dbsetup/training_data2.csv', 'r') as f:
+        reader = csv.reader(f)
+        next(f)  # skipping the header row
+        # order: id 6, link 13
+        data = []
+        for row in reader:
+            # Converts strings like this:
+            # ['a', 'b', 'c']
+            # ..into a Python list of strings
```
Comment (michhottinger) [inline dbsetup/seed/evidences.py:15]: I like the comments, but consider taking them out if they are obvious.

File: dbsetup/seed/incidents.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                place_id_lookup[state_code] = {}
+            place_id_lookup[state_code][city] = pk
+        pprint(place_id_lookup)
+
+        # creates the dynamic data for the SQL query
+        data = []
+        for row in reader:
+            # may God forgive us
+            if row['id'] == 'or-orlando-5' or row['id'] == 'or-orlando-8':
+                data.append([row['id'], 67, row['text'], row['date']])
```
Comment (michhottinger) [inline dbsetup/seed/incidents.py:31]: Already forgiven, lol

**Review by michhottinger — COMMENTED** [review]
This is a lot of work in here--should feel pretty proud :). Looks great.

**Pull request merged**